documentation/docs/05-misc/01-faq.md

```diff
@@ -131,6 +131,17 @@ If you need hash-based routing on the client side, check out [svelte-spa-router]
 
 You can see a [community-maintained list of routers on sveltesociety.dev](https://sveltesociety.dev/components#routers).
 
+## Can I tell Svelte not to remove my unused styles?
+
+No. Svelte removes the styles from the component and warns you about them in order to prevent issues that would otherwise arise.
+
+Svelte's component style scoping works by generating a class unique to the given component, adding it to the relevant elements in the component that are under Svelte's control, and then adding it to each of the selectors in that component's styles. When the compiler can't see what elements a style selector applies to, there would be two bad options for keeping it:
+
+- If it keeps the selector and adds the scoping scoping class to it, there is no guarantee that the selector will match the expected elements in the component, and they definitely won't if they were created by a child component or `{@html ...}`.
```

Comment:
For some reason github won't let me make a suggestion here, but: 'there is no guarantee that the selector will match the expected elements' is incorrect, I think? it's guaranteed _not_ to match anything;

> If it keeps the selector and adds the scoping class to it, it won't match any elements inside the component

Also, I'm not sure if 'scoping scoping' was a reference to the fact that we sometimes add the scoping class twice

Reply:
It could theoretically match, I think, if the user programmatically added a matching class to an element that was already getting the scoping class applied for another reason. (This is also adjacent to what Dominik and I were discussing in Discord about vite-plugin-svelte's behavior.) Since this is the FAQ entry we'll point people to when they request this feature which seems quite reasonable on the surface, I wanted to be very careful about what I say is impossible versus just unlikely. I'm happy to change this to something else if you think that would be a good idea.

lol, 'scoping scoping' is in fact a copyediting failure. Will fix.

Reply:
I tweaked "no guarantee" to "likely not". Will go ahead and merge now. Feel free to tweak or revert if you disagree with my change